package.json

```diff
@@ -145,11 +148,10 @@
   },
   "lint-staged": {
     "*.js": [
-      "eslint --fix",
+      "format",
```

Comment:
Will this format/lint the whole codebase before each commit?
I think what we rather want here is to replace this with
```
"prettier --write",
"eslint --fix"
```
... so that only staged files are formatted.

Reply:
Hmm.. I just implemented this change locally, and now neither solution is running the lint-staged scripts on commit. Here's what I'm doing:

1. Make a change in a file in the `src` folder that's nonconformant to our prettier rules (like adding an extra tab or a really long line).
2. `git add src && git commit -m "This should run the 'format' command"`
3. I see the files immediately committed, and nothing from `husky`.

I've clearly gone mad, so I'm gonna go to bed :P. Is anybody else seeing this behavior?